services/plugin/src/Plugin.tsx

```diff
@@ -118,68 +113,81 @@ export const Plugin = ({
+    // send them to the plugin
     useEffect(() => {
-        // if communicationReceived switches from false to true, the props have been sent
-        const prevCommunication = prevCommunicationReceived
-        setPrevCommunicationReceived(communicationReceived)
-        if (prevCommunication === false && communicationReceived === true) {
+        if (!iframeRef.current) {
             return
         }
 
-        if (iframeRef?.current) {
-            const iframeProps = {
-                ...memoizedPropsToPass,
-                alertsAdd,
-                // If a dimension is either specified or container-driven,
-                // don't send a resize callback to the plugin. The plugin can
-                // use the presence or absence of these callbacks to determine
-                // how to handle sizing inside
-                setPluginHeight: !height ? setPluginHeight : null,
-                setPluginWidth: !width && clientWidth ? setPluginWidth : null,
-                setInErrorState,
-                setCommunicationReceived,
-                clientWidth,
-            }
-
-            // if iframe has not sent initial request, set up a listener
-            if (!communicationReceived && !inErrorState) {
-                const listener = postRobot.on(
-                    'getPropsFromParent',
-                    // listen for messages coming only from the iframe rendered by this component
-                    { window: iframeRef.current.contentWindow },
-                    (): any => {
-                        setCommunicationReceived(true)
-                        return iframeProps
-                    }
-                )
-                return () => listener.cancel()
-            }
-
-            // if iframe has sent initial request, send new props
-            if (
-                communicationReceived &&
-                iframeRef.current.contentWindow &&
-                !inErrorState
-            ) {
-                postRobot
-                    .send(
-                        iframeRef.current.contentWindow,
-                        'updated',
-                        iframeProps
-                    )
-                    .catch((err) => {
-                        // log postRobot errors, but do not bubble them up
-                        console.error(err)
-                    })
-            }
+        const iframeProps = {
+            ...memoizedPropsToPass,
+            alertsAdd,
+            // If a dimension is either specified or container-driven,
+            // don't send a resize callback to the plugin. The plugin can
+            // use the presence or absence of these callbacks to determine
+            // how to handle sizing inside
+            setPluginHeight: !height ? setPluginHeight : null,
+            setPluginWidth: !width && clientWidth ? setPluginWidth : null,
+            setInErrorState,
+            clientWidth,
```

Comment:
`setCommunicationReceived` was removed here, since it wasn't needed for successfully reloading a broken plugin -- here's a test I did in the Simple App in the app platform, using the `Alerter` as a plugin:


https://github.com/user-attachments/assets/549a4246-e7fb-4dac-a9c1-06d164d532a0